engine/apps/grafana_plugin/views/status.py

```diff
@@ -42,7 +33,6 @@ def get(self, request: Request) -> Response:
         return Response(
             data={
                 "is_installed": is_installed,
-                "grafana_connection_ok": connected_to_grafana,
```

Comment:
this attribute is not consumed by the frontend. therefore I propose to get rid of it ([repo search results](https://github.com/grafana/oncall/search?q=grafana_connection_ok))

Reply:
@maskin25

Reply:
It was going to provide status in cloud for if the URL from plugin provisioning wasn't working but since we aren't using it might as well remove it.